python/src/aac/context/definition_parser.py

```diff
@@ -829,5 +829,6 @@ def load_definitions(
             self.context.context_instance.fully_qualified_name_to_definition[
                 f"{definition.package}.{definition.name}"
             ] = definition
+            definition.source.is_loaded_in_context = True
```

Comment:
Also, this line actually needs to take place at line 827. Otherwise the returned result does not have a concept of this update nor does the definition loaded into context. You are updating the flag after both data structures have already been provided a copy of the definition object.